### ISSUE TITLE
suggestion, read the suggested address in the address bar chrome

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#5703]
If you type in a new address in Chrome's address bar, the site of suggestion is not read by NVDA.
 palaces to play:
1. open the chrome
2. press ctrl + l
3. enter a site, such www.nvda-project.org
   expected that when you enter the www.nv NVDA read the hint of chrome that is www.nvda-project.org
   What happens is that NVDA does not read the suggestion, but read deselected w deselected.  w deselected  w deselected  n deselected v

**Comment by leonardder** [comment]
I can't reproduce. When i start typing nv in chrome, NVDA says "access.org selected instead"

@fernando-jose-silva, how does this work for you nowadays?

**Comment by fernando-jose-silva** [comment]
It's perfect, you're reading today, thank you very much.
